astro/src/content/docs/apis/_user-search-request-body-database-examples.mdx

```diff
@@ -6,4 +6,7 @@ import JSON from 'src/components/JSON.astro';
 
 <JSON title="Example Request JSON searching by queryString for users with a firstName, lastName, fullName, email or username containing piper." src="users/search-queryString-match-example-database-request.json" />
 
+<JSON title="Example Request JSON searching by queryString for users with a phoneNumber."
```

Comment:
NOTE: This example does not work with Elastic. If we want to pick an example here that works for both, I can.

Reply:
This is for the database search section so seems fine.